docs/user_guide/flyte_fundamentals/registering_workflows.md

```diff
@@ -365,6 +365,15 @@ two GitHub actions that facilitates this:
   of Flyte packages, for example, the `.tgz` archives that are created by
   `pyflyte package`.
 
+## Some of the best practices for CI/CD
```

Comment:
It's important to add a disclaimer-like intro. Something like " In case your development process registers Flyte workflows on each commit, consider the following recommendations" or something along these lines. We don't want users to infer that this is the only way of doing CI/CD with Flyte.

Reply:
updated it